app/README.md

```diff
@@ -29,3 +29,6 @@ needs to be queried to get older stats. As a result, the `get_specific_template_
 `template/rest.py` relies on a method, `fetch_template_usage_for_service_with_given_template()`, that first
 queries the `ft_notification_status` table and then the `notifications` table if the user needs to get stats
 from today.
+
+## message_limit default
+The service called "BVA" was created in the staging environment to update the message_limit of all services from 1000 to 100.
```

Comment:
Is this specific to staging?

Reply:
The BVA service was created in the Dev environment, message_limit applies to the service.